song-server/src/main/java/bio/overture/song/server/service/FileModificationService.java

```diff
@@ -93,46 +91,26 @@ public FileUpdateResponse securedFileWithAnalysisUpdate(
     // analysis
     val currentState = analysisService.readState(analysisId);
     checkServer(
-        currentState != SUPPRESSED,
+        currentState == UNPUBLISHED,
```

Comment:
Why do we want to let people move published data from open to controlled without changing the publication status?

Reply:
Because it does not affect the integrity of the data (i.e md5 and size). The updateFile endpoint can only be used by an authorized user for that study. Also, md5 and size are the only thing that affect score.